cmake/Boost_DD.cmake

```diff
@@ -2,10 +2,9 @@
 # Loads the boost library giving the priority to the system package first, with a fallback
 # to the submodule.
 #
-include_guard(GLOBAL)
-
-find_package(Boost 1.85)
+find_package(Boost 1.85 QUIET)
```

Comment:
Why quiet?

Reply:
If the package is not found it prints messages whenever the file is included. These are annoying, especially when there no version of the said package in the system as the warnings are quite long.

I thought quiet option is acceptable in this case as we are printing our custom message once anyway...

Reply:
Sounds fine to me, thanks!